### PR TITLE
Remove mention of TCE

### PR DESCRIPTION
**Pull request by marckhouzam**
## What this PR does / why we need it

The TCE product has been discontinued.  The PR removes the installation and mention of TCE.
This repo will now only allow installing the Tanzu CLI.

## Which issue(s) this PR fixes

Fixes: # N/A

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -4,10 +4,8 @@
 
 | Maintainer | GitHub ID | Affiliation |
 |------------|-----------|-------------|
-| Josh Rosso | [joshrosso](https://github.com/joshrosso) | [VMware](https://www.github.com/vmware/) |
-| Nicholas Seemiller | [seemiller](https://github.com/seemiller) | [VMware](https://www.github.com/vmware/) |
-| Sean McGinnis | [stmcginnis](https://github.com/stmcginnis) | [VMware](https://www.github.com/vmware/) |
-| David vonThenen | [dvonthenen](https://github.com/dvonthenen) | [VMware](https://www.github.com/vmware/) |
-| John McBride | [jpmcb](https://github.com/jpmcb) | [VMware](https://www.github.com/vmware/) |
-| Nolan Brubaker | [nrb](https://github.com/nrb) | [VMware](https://www.github.com/vmware/) |
+| Vuil Lam | [vuil](https://github.com/vuil) | [VMware](https://www.github.com/vmware/) |
```
Comment (vuil) [inline MAINTAINERS.md:7]: nit:
VMware by Broadcom?

**Review by vuil — APPROVED** [review]
Nice! Thanks for the cleanup!

File: MAINTAINERS.md (hunk 1 above)
Comment (marckhouzam) [reply MAINTAINERS.md:7]: Good point.  will do.  Do you think the link https://www.github.com/vmware/ is ok to use?

Comment (marckhouzam) [reply MAINTAINERS.md:7]: Forgot to push the changes to address this comment.  Did it in #61

**Pull request merged**